src/packages/health-check/dashboard-health-check.element.ts

```diff
@@ -58,7 +59,7 @@ export class UmbDashboardHealthCheckElement extends UmbLitElement {
 				weight: 500,
 				meta: {
 					label: group.name || '',
-					api: () => import('./health-check.context.js'),
+					api: UmbHealthCheckContext,
```

Comment:
Now that you are using the `loadManifestApi()` function, I think it should work if you leave this as a dynamic import, right?